peps/pep-0748.rst

```diff
@@ -0,0 +1,922 @@
+PEP: 748
+Title: A Unified TLS API for Python
+Author: Joop van de Pol <joop.vandepol@trailofbits.com>,
+        William Woodruff <william@yossarian.net>
+Sponsor: Alyssa Coghlan <ncoghlan@gmail.com>
+Discussions-To: https://discuss.python.org/t/pre-pep-discussion-revival-of-pep-543/51263
+Status: Draft
+Type: Standards Track
+Created: 27-Jun-2024
+Python-Version: 3.13
+Post-History: `17-Apr-2024 <https://discuss.python.org/t/pre-pep-discussion-revival-of-pep-543/51263>`__
+Replaces: 543
```

Comment:
We give PEP 543 a `Superseded-By` header pointing to 748.

https://peps.python.org/pep-0001/#pep-header-preamble

Reply:
Done, thank you!